EmployeeList/Persistence/OnDiskCache.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  Cache.swift
+//  EmployeeList
+//
+//  Created by Wang Siman on 1/29/21.
+//  Copyright © 2021 None. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import Combine
+
+final class Cache {
+    
+    private var fileManager: FileManager = FileManager.default
+    
+    private func buildFilePath(forKey key: String) -> URL? {
```

Comment:
if you don't have a documents directory, things have gone Horribly Wrong™, and you should probably crash. I would make this a non-optional.

Reply:
1. I wonder under which case there isn't a document directory. Does it frequently happen in real world?
2. What is the safe way to handle this exception without hurting user experience?
3. Could it be a general problem that build an invalid path in other storage system like database?

Reply:
There's never a case where you don't have ~/Documents.